client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -16,8 +16,10 @@ import { AssertConditionState } from './internals/types';
 import type { AssertConditionResult, Flow, ProvidedDependencies } from './internals/types';
 import type { OnboardSelect, SiteSelect, UserSelect } from '@automattic/data-stores';
 
+const FLOW_NAME = 'site-migration';
```

Comment:
What do you think about exporting this in case we need to refer to it from another flow?